USING.md

```diff
@@ -1,116 +1,210 @@
-## Using the CLI
+# Using the CLI
 
-### Invoking the command line tool
+## Table of contents
+* [Docker or local installation](#Docker-or-local-installation)
+* [Invoking the command line tool](#Invoking-the-command-line-tool)
+* [Test reports](#Test-reports)
+* [Translation]
+* [Advanced use](#Advanced-use)
 
-The most basic use of the `zonemaster-cli` command is to just test a domain
-like this:
 
-    $ zonemaster-cli example.com
+## Docker or local installation
 
-The output comes continously as the tests are performed.
+The `zonemaster-cli` tool can be run from the command line of any computer that
+meets one of the following requirements:
+
+* Docker is installed on the computer, or
+* Zonemaster-CLI has been installed on the computer.
 
-    Seconds Level     Message
-    ======= ========= =======
-       1.16 NOTICE    No illegal characters in the domain name (example.com).
-       2.01 WARNING   SOA 'refresh' value (10800) is less than the recommended one (14400).
-      13.86 CRITICAL  All nameservers are in the same AS (12345).
-      13.87 NOTICE    123.456.789.0 returned no DS records for example.com.
+### Using Docker
 
-If your machine is for some reason not configured for use with IPv6 you
-want to disable the use of IPv6 with the `--no-ipv6` option. If you want
-to see how your domain is configured to be used with no IPv4 available
-for the client there is also the `--no-ipv4` option - however, both cannot
-be used at the same time.
+To run Zonemaster-CLI on docker you have to make sure that Docker is installed.
```

Comment:
Is there a reason why this one reference to Docker isn't capitalized when all the others are?